name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/preferences/PPIDPreferencePage.java

```diff
@@ -28,17 +28,16 @@
 
 public class PPIDPreferencePage extends PreferencePage
 {
+    private static final String EMPTY_USER_TEXT = "-"; //$NON-NLS-1$
     private final OAuthClient oauthClient = OAuthClient.INSTANCE;
-
     private Label user;
     private Button action;
-
-    private Runnable updateListener = () -> Display.getDefault().asyncExec(this::triggerUpdate);
+    private final Display display = Display.getDefault();
```

Comment:
It's okay to use this as a member variable, but generally I prefer the direct use of the static method unless the display is actually context dependent (e.g. for the window management). Less instance variables.